### ISSUE TITLE
Submit button enabled for Android on "Enter your 8-digit number" window

### ISSUE THREAD

**Issue by mmiron-HC** [cds-snc/covid-shield-mobile#252]
Android v9
Samsung Galaxy S8+
both languages
From menu, under the Diagnosed with COVID-19 section press the Enter number button
From the Enter your 8-digit number window, press again Enter number button
the Enter your 8-digit number window appears. 
Submit is enabled. Click Submit - error message appears. 
According to Julianna, this behaviour should be the same as iOS. Submit button should be disabled until all 8 digits are entered.
![Android_submit enabled](https://user-images.githubusercontent.com/65728700/85945692-8bd61d00-b90d-11ea-8e5f-bd9fb05bf87e.jpg)

**Comment by its-anirudh** [comment]
Hi @JuliannaR @mmiron-HC , I could not replicate this issue. The submit button gets activated only upon all the numbers being entered.

![image](https://user-images.githubusercontent.com/65962820/86819775-22b18080-c056-11ea-8b66-c479498e0efb.png)


![image](https://user-images.githubusercontent.com/65962820/86819838-352bba00-c056-11ea-82c4-2ea156f7d155.png)

**Comment by JuliannaR** [comment]
@its-anirudh did you test with android or IOS?

**Comment by its-anirudh** [comment]
> @its-anirudh did you test with android or IOS?

Android

**Comment by JuliannaR** [comment]
@mmiron-HC can you confirm if this is still an issue for you? - Thanks!

**Comment by mmiron-HC** [comment]
Using the OnePlus 3, the button looks disabled, however if I click on it it "thinks" and then prompts me with the Number not recognized prompt. 
![IMG_1795](https://user-images.githubusercontent.com/65728700/87029109-ac279680-c1ad-11ea-989c-3fed4c71372e.PNG)
![IMG_1796](https://user-images.githubusercontent.com/65728700/87029120-b184e100-c1ad-11ea-8cfc-95437a0f788c.PNG)

**Comment by timarney** [comment]
Thanks for confirming.

**Comment by its-anirudh** [comment]
So @timarney is this an issue ?

**Comment by timarney** [comment]
We need to determine - if it's an expired code or a bug.

Can you test on your end.  I can submit a code on mine.

**Comment by mmiron-HC** [comment]
Note that this has nothing to do with the code. I enter none. I simply click on submit.

**Comment by its-anirudh** [comment]
> We need to determine - if it's an expired code or a bug.
> 
> Can you test on your end. I can submit a code on mine.

I think i know what the issue is...the submit button is still clickable when grey.

**Comment by its-anirudh** [comment]
Pushed Pr

**Comment by mmiron-HC** [comment]
Confirmed, the Submit button is now truly disabled.
